modules/Microsoft.Compute/virtualMachines/readme.md

```diff
@@ -1822,7 +1822,7 @@ module virtualMachines './Microsoft.Compute/virtualMachines/deploy.bicep' = {
       ]
     }
     extensionCustomScriptProtectedSetting: {
-      commandToExecute: '<commandToExecute>'
+      commandToExecute: 'powershell -ExecutionPolicy Unrestricted -Command \'& ./${resourceGroupResources.outputs.storageAccountCSEFileName}\''
```

Comment:
Wondering if it wouldn't be more intuitive to show the actual cmdlet instead of a generic token in this case. Thoughts?

Reply:
Lke discussed - in general this is almost always true, but we have to tackle this on a more fundamental level (i.e. find a meaningful way to provide the content - ideally without showing template code). 
To this end I'd strongly suggest to keep it out of this PR

Reply:
🙆‍♀️